data/glsk/glsk-quality-check-ucte/pom.xml

```diff
@@ -10,7 +10,7 @@
         <version>2.2.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>farao-glsk-quality-check</artifactId>
+    <artifactId>farao-glsk-quality-check-ucte</artifactId>
```

Comment:
Could this quality check be independent of the source format with new structure or not ?

Reply:
Agreed that we can refactor this part later on